lib/screens/login_screen.dart

```diff
@@ -2,13 +2,45 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:scdao_mobile/constants/color_constants.dart';
+import 'package:http/http.dart' as http;
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'dart:convert';
+import 'dart:io';
 
-class LoginScreen extends StatefulWidget {
+class LoginScreen extends ConsumerStatefulWidget {
   @override
   _LoginScreenState createState() => _LoginScreenState();
 }
 
-class _LoginScreenState extends State<LoginScreen> {
+final loginRepositoryProvider =
+    Provider<LoginRepository>((ref) => LoginRepository());
+
+class LoginRepository {
+  var url = Uri.parse("http://192.168.0.44:8888/api/token");
```

Comment:
Please make this set from an environment file or somewhere else central so that we can change the URL in a single place.

Reply:
This is one way of doing it: https://github.com/BU-Spark/se-scdao-mobile-flutter/blob/ian-updates/lib/constants/api_constants.dart